dependencies_test.go

```diff
@@ -17,11 +17,11 @@ func TestCreateServiceDependency(t *testing.T) {
 	client := BestTestClient(t, "serviceDependencyCreate", testRequest)
 	// Act
 	result, err := client.CreateServiceDependency(ol.ServiceDependencyCreateInput{
-		Key: ol.ServiceDependencyKey{
-			Service:   *ol.NewIdentifier("example_2"),
-			DependsOn: *ol.NewIdentifier("example_3"),
+		DependencyKey: ol.ServiceDependencyKey{
```

Comment:
WTF - why are these optional.  🤦 our API is soo terrible sometimes.  functionality wise you need both 😮‍💨

Reply:
We could add an exception in the generator to force that :trollface:

Reply:
Na thats ok just realizing how bad our API is with optional values.